app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,80 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?   
+	  erb :'tweets/index'
+	else
+	  redirect to '/login'
+	end  
+  end
+
+  get '/tweets/new' do
+  	if logged_in?
+  	  erb :'tweets/new'
+  	else
+  		redirect to '/login'
+  	end  
+  end
+
+  post '/tweets' do
+    @user = User.find_by_id(session[:user_id])
+    if params[:content] == ""
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.